solidity/contracts/IConversionPathFinder.sol

```diff
@@ -1,9 +1,10 @@
-pragma solidity 0.4.26;
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity >=0.6.12 <0.7.0;
 import "./token/interfaces/IERC20Token.sol";
 
 /*
     Conversion Path Finder interface
 */
-contract IConversionPathFinder {
-    function findPath(address _sourceToken, address _targetToken) public view returns (address[] memory);
+abstract contract IConversionPathFinder {
```

Comment:
Why not `Interface`?

Reply:
there might be more I didn't change even though I could, though for this specific one, I believe it's a recursive function so it can't be external

Reply:
No, it's the private function `function getPath(address _token, IConverterRegistry _converterRegistry)`, which is recursive.
So you can use an `interface` here.

Reply:
updated

Reply:
I'd try to change to an `Interface` anyway. It should work 95% of the times.

Reply:
yeah I did change that one

Reply:
come to think of it, we can also do that for the formula since it seems like public functions can override external interfaces but that seems like something that will get fixed at some point